setup.py

```diff
@@ -213,7 +213,7 @@ def run(self):
     'pysha3==0.3',
     'pytest==2.9.2',
     'pytest-cov==2.2.1',
-    'python-bitcoinlib==0.7.0',
+    'python-bitcoinlib', #==0.7.0
```

Comment:
By removing a specific version constraint, does this mean that the python-bitcoinlib version for different installs of counterparty based on the time that the installation was performed?

If so, we should lock this to 0.7.0 until we can test a newer version.